DietPi-Software | Koel: Move install dir to dietpi_userdata

MichaIng opened the pull request:
**Status**: Testing
- [x] Patch Koel directory on DietPi-Update
- [x] Create phpinfo.php opcache.php and apc.php only, if webserver is installed
- [x] Changelog entry

- [x] 🈯️ Test install on Stretch VM
- [x] 🈯️ Testing move of Koel install dir on patch

**Reference**: https://github.com/Fourdee/DietPi/pull/1948

**Commit list/description**:
+ DietPi-Software | Koel: As it is not accessed via webserver, move "koel" to dietpi_userdata and do not add it to www-data group
+ DietPi-Software | Koel: Remove duplicated ".env" setting injection
+ DietPi-Patchfile | Move Koel from /var/www to /mnt/dietpi_userdata, as it is not served via webserver
+ DietPi-Software | MariaDB: Fix Jessie uninstall issues; Remove hidden user config dir

MichaIng committed:
v6.13

+ DietPi-Software | Koel: As it is not accessed via webserver, move "koel" to dietpi_userdata and do not add it to www-data group
+ DietPi-Software | Remove duplicated ".env" setting injection

MichaIng committed:
v6.13

+ DietPi-Software | PHP: Create PHP info pages only, if webserver was installed

MichaIng committed:
v6.13

+ DietPi-Patchfile | Move Koel from /var/www to /mnt/dietpi_userdata, as it is not server via webserver

MichaIng committed:
v6.13

+ CHANGELOG | Moved Koel install dir to /mnt/dietpi_userdata

MichaIng committed:
v6.13

+ DietPi-Patchfile | Koel: Chown $G_FP_DIETPI_USERDATA/koel accordingly
+ DietPi-Patchfile | Koel: Patch koel.service accordingly

MichaIng commented on dietpi/dietpi-software:
```diff
@@ -3301,7 +3301,7 @@ _EOF_
 			[[ $target ]] && target="-d $target"
 			l_message='Unzipping archive' G_RUN_CMD unzip -o $file $target
 
-		elif [[ $type == tar ]]; then # .bz2 .gz may require different extraction vars if I remember correctly?
```
@Fourdee 
I tested it on all so far affected installs: `tar` uses `gzip`/`bzip2` module very well by itself. The additional arguments are needed only for packing, not for unpacking.

Fourdee commented:
> DietPi-Patchfile | Move Koel from /var/www to /mnt/dietpi_userdata, as it is not server via webserver

@MichaIng 

Nice one 👍 Much better location for it.

Fourdee reviewed (approved):
Looks good 👍 

Merge when required.

MichaIng committed:
v6.13

+ DietPi-Software | MariaDB: Fix Jessie uninstall issues; Remove hidden user config dir

MichaIng committed:
v6.13

+ DietPi-Software | MariaDB: Apply uninstall fix on Jessie only
